### ISSUE TITLE
log files

### ISSUE THREAD

**Issue by evagian** [crisb-DUT/DnCNN-tensorflow#30]
First of all great job! 

I have a question about the print outputs (ex Epoch: [10] [2327/5198] time: xxx, loss: xxx)... 

Are they being saved somewhere (in the log files for example)? And if yes, how can I access them?

I want to use them for plotting the train vs dev PSNR for every epoch. 

Thank you!

**Comment by crisb-DUT** [comment]
Hi @evagian ,

The loss is plotted by tensorboard, you can view it in your browser. And you can also add anything you want to plot to the tensorboard summary as the loss plotting do. The log file can be downloaded from tensoirboard in json format.

Thanks,
Wenbo

**Comment by evagian** [comment]
Thank you very much ! :)
